recipes/fit-sne/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = 'FIt-SNE' %}
+{% set version = '1.1.0' %}
+{% set sha256 = '9a985fe1bb1e9c17fff0e81968e794eb834ccf002695d02294f2ff9ac564b873' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KlugerLab/FIt-SNE/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+  host:
+    - fftw
+
+test:
+  commands:
+    - fast_tsne 2>&1 | grep 'Done'
+
+about:
+  home: https://github.com/KlugerLab/FIt-SNE
+  license: Multiple
+  license_file: LICENSE.txt
```

Comment:
Also, you need a licence_family entry

Reply:
@scopatz This particular package includes license from multiple sources -- what would be an appropriate value for for license_family and license in such a case? Thanks!

Reply:
I used "OTHER" based on my review of conda-build source -- let me know if that's not appropriate.

Reply:
That's fine